### PR TITLE
Solution from 2021-06-15 Meetup, in Ruby

### PR DESCRIPTION
**Pull request by ericwburden**
Cleaned up a bit after the meetup, still probably not super idiomatic.

### CODE REVIEW

File: challenges/2021-06-15-square-maze/solutions/ruby/mob/mob.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+
+def can_move_down?(position)
+  position.first < @final_row_index
+end
+
+
+def can_move_right?(position)
+  position.last < @final_col_index
+end
```
Comment (parhaml) [inline challenges/2021-06-15-square-maze/solutions/ruby/mob/mob.rb:34]: nice abstractions. It makes the main method easier to grok

**Review by parhaml — APPROVED** [review]
Textbook ruby. 💯 would not think you just picked it up.

File: challenges/2021-06-15-square-maze/solutions/ruby/mob/mob.rb (hunk 1 above)
Comment (ericwburden) [reply challenges/2021-06-15-square-maze/solutions/ruby/mob/mob.rb:34]: Aw, shucks...

**Review by birenderjit — APPROVED** [review]
I agree with Lee, Like the way you have named stuff.

**Pull request merged**